### PR TITLE
Feature/redlight behavior

### PR DESCRIPTION
**Pull request by Jessica-vdz**
whats new:
- i've added the behavior for the redlight block

how to test:
- try setting the boolean variable to true of false 

additional information:
- none

### CODE REVIEW

File: src/main/java/nl/jessicavdz/redlightgreenlight/Block/BlockBreakCancel.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockBreakEvent;
+
+public class BlockBreakCancel implements Listener {
+
+    @EventHandler
+
+    public void onBlockBreak(BlockBreakEvent event) {
+        Player player = event.getPlayer();
+        if(player.isOp()) return;
```
Comment (Stanley-Dam) [inline src/main/java/nl/jessicavdz/redlightgreenlight/Block/BlockBreakCancel.java]: It is better not use return here, but rather check if the player is not OP (!player.isOp()) and than execute the setcancelled if that's the case

File: src/main/java/nl/jessicavdz/redlightgreenlight/RedLightGreenLight.java [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 
     @Override
     public void onEnable() {
         // Plugin startup logic
+        RedLightBehavior redLightBehavior = new RedLightBehavior();
 
-    }
+        getServer().getPluginManager().registerEvents(new BlockBreakCancel(),this);
+        getServer().getPluginManager().registerEvents(redLightBehavior, this);
+        redLightBehavior.light = false;
```
Comment (Stanley-Dam) [inline src/main/java/nl/jessicavdz/redlightgreenlight/RedLightGreenLight.java]: This is temporary testing code, this should not get pushed to develop.

**Review by Stanley-Dam — COMMENTED** [review]
Very nice work!

File: src/main/java/nl/jessicavdz/redlightgreenlight/Block/BlockBreakCancel.java (hunk 1 above)
Comment (Jessica-vdz) [reply src/main/java/nl/jessicavdz/redlightgreenlight/Block/BlockBreakCancel.java]: i've fixed it. thanks!

File: src/main/java/nl/jessicavdz/redlightgreenlight/RedLightGreenLight.java (hunk 2 above)
Comment (Jessica-vdz) [reply src/main/java/nl/jessicavdz/redlightgreenlight/RedLightGreenLight.java]: whoops

Comment (Jessica-vdz) [reply src/main/java/nl/jessicavdz/redlightgreenlight/RedLightGreenLight.java]: fixed

**Pull request merged**